CHANGELOG.md

```diff
@@ -3,7 +3,10 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Sync Breaking Changes (In Beta)
 
-* None.
+* Underlying sync engine upgraded to version BETA-6.3.
```

Comment:
Reminder: this needs to be updated to whatever actual version ends up being pulled into this commit.